Guide.docc/IncrementalAdoption.md

```diff
@@ -149,6 +149,60 @@ class UIStyler {
 Combining static and dynamic isolation can be a powerful tool to keep the
 scope of changes gradual.
 
+## Missing Annotations
+
+Dynamic isolation gives you tools to express isolation at runtime.
```

Comment:
Hmmm, but the example below is static isolation.

Static == compile time; dynamic == stuff runtime knows but compiler doesnt

Reply:
Yes exactly! This is a contextual sentence that leads into the next to descibe what is actually going to be covered. Does that read ok?

Reply:
Aha you're leading from the previous section a bit... yeah let's keep and worst case we revisit :) 

resolving